pom.xml

```diff
@@ -1836,7 +1836,7 @@
                 <module>tests/ballerina-compiler-plugin-test</module>
                 <module>cli/ballerina-cli-utils</module>
 
-                <module>language-server</module>
+                <!--<module>language-server</module>-->
```

Comment:
@rdhananjaya Hope this was added mistakenly.

Reply:
Yes yes, never intended to commit a pom here. Will remove